cli/src/alluxio.org/cli/cmd/exec/test_hdfs_mount.go

```diff
@@ -41,26 +43,35 @@ func (c *TestHdfsMountCommand) Base() *env.BaseJavaCommand {
 
 func (c *TestHdfsMountCommand) ToCommand() *cobra.Command {
 	cmd := c.Base().InitRunJavaClassCmd(&cobra.Command{
-		Use:   "testHdfsMount",
+		Use:   "hdfsMountTest",
 		Args:  cobra.NoArgs,
 		Short: "Tests runs a set of validations against the given hdfs path.",
 		RunE: func(cmd *cobra.Command, args []string) error {
 			return c.Run(args)
 		},
 	})
 	cmd.Flags().StringVar(&c.path, "path", "",
-		"[required] specifies the HDFS path you want to validate.")
+		"specifies the HDFS path you want to validate.")
 	cmd.Flags().BoolVar(&c.readonly, "readonly", false,
 		"mount point is readonly in Alluxio.")
 	cmd.Flags().BoolVar(&c.shared, "shared", false,
 		"mount point is shared.")
 	cmd.Flags().StringVar(&c.option, "option", "",
 		"options associated with this mount point.")
+	err := cmd.MarkFlagRequired("path")
+	if err != nil {
```

Comment:
the error here could be ignored as the manual suggests: https://github.com/spf13/cobra/blob/v1.7.0/user_guide.md#required-flags
the call here is just marking the flag, not parsing it. so if the flag is missing, the actual error would be triggered and handled by the cobra code.

Reply:
agree. in almost all situation, the returned error should be checked, but this is one of the rare exceptions

Reply:
Understood. I will remove all error checks